Azure/sonic-buildimage
# [device] Add as4630_54pe SDK configuration parameters

**- Why I did it**
- Add as4630_54pe SDK configuration parameters.

**- How I did it**
- Add  l3_alpm_enable=2 and ipv6_lpm_128b_enable=1 in hx5-as4630-48x1G+4x25G+2x100G.bcm.

**- How to verify it**
- Compile SONiC source with this paramters to confirm building process.
- Running SONiC image with this change items on as4630_54pe.

**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
